doc/02-configuration.md

```diff
@@ -34,23 +34,26 @@ Setting a custom POST path can be useful in avoiding adblockers; it replaces the
+| `base64Encoding`        | `bool?` | Indicates whether payload JSON data should be base64 encoded.                                                                                                                     | ✔ | ✔ | ✔ | true                                          |
+| `platformContext`       | `bool?` | Indicates whether [platform](http://iglucentral.com/schemas/com.snowplowanalytics.snowplow/mobile_context/jsonschema/1-0-2) (mobile) entity should be attached to tracked events. | ✔ | ✔ | | true                                          |
+| `geoLocationContext`    | `bool?` | Indicates whether [geo-location](http://iglucentral.com/schemas/com.snowplowanalytics.snowplow/geolocation_context/jsonschema/1-1-0) entity should be attached to tracked events. | ✔ | ✔ | ✔ | false                                         |
+| `sessionContext`        | `bool?` | Indicates whether [session](http://iglucentral.com/schemas/com.snowplowanalytics.snowplow/client_session/jsonschema/1-0-2) entity should be attached to tracked events.           | ✔ | ✔ | ✔ | true                                          |
+| `webPageContext`        | `bool?` | Indicates whether context about current [web page](http://iglucentral.com/schemas/com.snowplowanalytics.snowplow/web_page/jsonschema/1-0-0) should be attached to tracked events. | | | ✔ | true                                          |
+| `screenContext`         | `bool?` | Indicates whether [screen](http://iglucentral.com/schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0) entity should be attached to tracked events.                      | ✔ | ✔ | | true                                          |
+| `applicationContext`    | `bool?` | Indicates whether [application](http://iglucentral.com/schemas/com.snowplowanalytics.mobile/application/jsonschema/1-0-0) entity should be attached to tracked events.            | ✔ | ✔ | | true                                          |
+| `webActivityTracking`   | WebActivityTracking?` | Enables activity tracking using page views and pings on the Web.                                                                                                                  | | | ✔ | true                                          |
+| `userAnonymisation`     | `bool?` | Anonymises certain user identifiers.                                                                                                                                              | ✔ | ✔ | ✔ | false                                         |
+| `lifecycleAutotracking` | `bool?` | Indicates whether the [lifecycle](iglu:com.snowplowanalytics.mobile/application_lifecycle/jsonschema/1-0-0) entity and events should be autotracked.                              | ✔ | ✔ | | false                                         |
```

Comment:
```suggestion
| `lifecycleAutotracking` | `bool?` | Indicates whether the [lifecycle](iglu:com.snowplowanalytics.mobile/application_lifecycle/jsonschema/1-0-0) entity and foreground and background events should be autotracked (only on iOS and Android). | ✔ | ✔ | | false |
```

Reply:
iOS and Android are indicated already by the table columns so I didn't add this change